Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -56,7 +56,9 @@ def runSelected(opt):
                      136.731, #2016B Photon data
                      140.53, #2011 HI data
                      1330, #Run2 MC Zmm
-                     135.4 #Run 2 Zee ttbar
+                     135.4, #Run 2 Zee ttbar
+                     11024.0, #2023LReco ttbar (w/ HGCal)
+                     11224.0  #2023GReco ttbar (flat tracker, Run2 calo)
```

Comment:
I was ruminating on this.
I think it's better to add both the pair of 2023 and the 2017 here in the short matrix explicitly.
These are reasonably short workflows (as long as you can submit them all in parallel).